djangocms_navigation/admin.py

```diff
@@ -46,9 +52,22 @@ def url_for_result(self, result):
 
 class MenuContentAdmin(admin.ModelAdmin):
     form = MenuContentForm
+    ItemModel = MenuItem
     list_display = ["title", "get_menuitem_link", "get_preview_link"]
     list_display_links = None
 
+    def get_admin_name(self, name, reverse=False, model=None):
```

Comment:
Why does this method have the optional model and the same method in MenuItemAdmin does not?

Reply:
It would be better to create a utility I think, call it something like get_model_admin_reverse in this projects utils.py like: https://github.com/divio/django-cms/blob/develop/cms/utils/urlutils.py#L95